src/screens/monitor/components/sourcesSelector/index.tsx

```diff
@@ -21,56 +12,66 @@ interface SelectorProps {
   onMenuClose: () => void;
 }
 
-export const sources_of_interest_list = ['camera_deputatilor', 'mfinante', 'mmediu', 'mdezvoltarii', 'meducatiei', 'mtransport', 'mai', 'mae', 'mapn', 'mjustitiei', 'senat_pl', 'camera_deputatilor_pl'];
+const sources_of_interest_list = [
+  'camera_deputatilor',
+  'mfinante',
+  'mmediu',
+  'mdezvoltarii',
+  'meducatiei',
+  'mtransport',
+  'mai',
+  'mae',
+  'mapn',
+  'mjustitiei',
+  'senat_pl',
+  'camera_deputatilor_pl',
+];
+
+const translatedSources = sources_of_interest_list.map((source) => Translations[source]);
+
+const translationToValueMap = sources_of_interest_list.reduce((map, source) => {
+  const translation = Translations[source];
+  map[translation] = source;
+  return map;
+}, {} as { [key: string]: string });
```

Comment:
```suggestion
}, {} as Record<string, string> });
```

Reply:
Record<type, type> is a built in typescript utility for this exact case { [key: string]: string }